tests/api_tests/nodes/test_views.py

```diff
@@ -1352,7 +1467,7 @@ def test_filtering_on_invalid_field(self):
         assert_equal(res.status_code, 400)
         errors = res.json['errors']
         assert_equal(len(errors), 1)
-        assert_equal(errors[0]['detail'], 'Query string contains an invalid filter.')
+        assert_equal(errors[0]['detail'], "'invalid' is not a valid field for this endpoint.")
```

Comment:
Not sure on this. On one hand, including the invalid field name is more user-friendly. On the other hand, including any user input is prone to developer error, and it's safer and easier to enforce a policy of excluding any user input from our responses. @brianjgeiger Thoughts on this? We can make a quick decision here, or I open a ticket if we want to defer discussion.